lib/src/codepage.dart

```diff
@@ -277,6 +277,25 @@ CodePageDecoder _createDecoder(String characters) {
   return _NonBmpCodePageDecoder._(result);
 }
 
+/// An input Sink for decoders where each input byte can be be considered
+/// independantly.
+class _CodePageSink implements Sink<List<int>> {
```

Comment:
This is a decoder sink, there should probably also be an encoder sink (a `StringConversionSink`), even if not today.
So name this something including `Decoder`.

Reply:
Done